tests/ControlledVocabulary.Tests/CvParamTests.fs

```diff
@@ -30,6 +34,13 @@ module InstanceMemberTests =
         let actual = testCvParams |> List.map (fun x -> x.Value)
         Assert.Equal<ParamValue List>(expected, actual)
 
+    [<Fact>]
+    let ``Equals`` () =
+        let testCvp1 = CvParam("test", "test", "test", ParamValue.Value "test", Generic.Dictionary<string,IParam>() |> fun d -> d.Add("test", testCvParams.Head); d) 
+        let testCvp2 = CvParam("test", "test", "test", ParamValue.Value "test", Generic.Dictionary<string,IParam>() |> fun d -> d.Add("test", testCvParams.Head); d) 
```

Comment:
Can you add some tests for CvParams with attributes? I am thinking of one where we expect them to be equal, and one where we expect them not to be equal

Reply:
Sorry, clarification: Attributes where the attributes themselves have attributes